packages/swingset-xsnap-supervisor/lib/capdata.js

```diff
@@ -0,0 +1,27 @@
+// This file is only used to generate the published bundle, so
+// @endo/init is in devDependencies (and this package should have no
+// direct dependencies), but eslint doesn't know that, so disable the
+// complaint.
+
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Let's just place an `.eslintrc` file in this directory which disables the rule for all files:

```json
{
    "rules": {
        "import/no-extraneous-dependencies": "off"
    }
}
```